### ISSUE TITLE
Warning: React.createClass is no longer supported. Use a plain JavaScript class instead.

### ISSUE THREAD

**Issue by anhtuank7c** [viromedia/viro#37]
I use React Native 0.47.2.

![screen shot 2017-10-05 at 12 34 56 am](https://user-images.githubusercontent.com/3163410/31190408-3173e8c0-a965-11e7-9ad4-9ddcb4048ce6.png)

**Comment by VikAdvani** [comment]
Hi @anhtuank7c, is this blocking you? You should see a yellow warning screen but still be allowed to proceed in debug or when using the testbed app. We are working on supporting create-react-class in our next release. When you build to release you should no longer see this warning in your app. Let us know, thanks!

**Comment by anhtuank7c** [comment]
@VikAdvani 
It's warning, i still playing well.
The warning shown while debugging app (I create new app from react-native init, then manually integrate react-viro)
Yes of course, the message will not show after release, but some next React version, they will completely remove this method.
You all should make this up to date.
Thank you

**Comment by anhtuank7c** [comment]
I wonder where is `react-viro` repo, can you show me.
I want to make it up to date to match my project requirement.
Thanks

**Comment by VikAdvani** [comment]
Hi @anhtuank7c,  the react-viro repo isn't open, but we plan to fix the issue with our 2.1 release in early November :)

**Comment by anhtuank7c** [comment]
@VikAdvani Thank for your information.
I hope it can work well on `React 16+`

**Comment by snowzurfer** [comment]
Hi @VikAdvani , when would the next release be? I have started using react-native recently, and have tried to generate a MWE but because of the changes to react-native and the problem above it does not work. Here is my code:

### App.js
```
import React, { Component } from 'react';
import { StyleSheet } from 'react-native';
import {
  ViroARScene,
  ViroARSceneNavigator,
  ViroText,
} from 'react-viro';

export class ARScene extends Component {
  constructor(props) {
    super(props);
    this.state = {text: 'Initializing ProvAR...'};
  }

  render() {
    return (
      <ViroARScene onTrackingInitialized={()=>{
        this.setState({text: 'Hello World'});
      }}>
        <ViroText text={this.state.text} scale={[.5, .5, .5]}
          position={[0, 0, -1]} style={styles.textStyle} />
      </ViroARScene>
    );
  }
}

export default class App extends Component {
  render() {
    return (
      <ViroARSceneNavigator apiKey='MY_KEY_HERE'
        initialScene={{scene: ARScene}} />
    );
  }
}
```
### index.js
```
import { AppRegistry } from 'react-native';
import App from './App';

AppRegistry.registerComponent('provar', () => App);
```

**Comment by dthian** [comment]
Hi @anhtuank7c, this should have been fixed in the latest release of Viro 2.1. 
To update, you can view the instructions [https://docs.viromedia.com/docs/releases](here). 
Please give it a shot and let us know if you encounter any issues. Thanks!

**Comment by anhtuank7c** [comment]
Thank you so much @dthian 👍
